### PR TITLE
Earth 1327 ally issues profile

### PR DESCRIPTION
**Pull request by aaroncole**
# READY FOR REVIEW

# Summary
- Replacing the definition list markup with header and unordered list to make it more a117 on the profile page.

# Needed By (Date)
- Next push

# Steps to Test

1. Navigate to /people/ines-azevedo
2. Inspect markup for the Email and Links section
3. Verify the markup uses an H2 and UL.
4. Compare to PROD.  They should look the same visually.

# Associated Issues and/or People
- JIRA ticket SE3:  EARTH-1327 (A11y issues with Profile markup)
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

**Review by buttonwillowsix — APPROVED** [review]
This looks visually fine.
@aaroncole the UL without LI feels a little weird to me? But it does seem to work fine.

**Comment by aaroncole** [comment]
@buttonwillowsix What do you mean by "UL without LI"?  Where do you see one?

File: templates/stanford_person/stanford-person.html.twig [hunk 1]
```diff
@@ -11,37 +11,38 @@
...
+
         {% endif %}
         {% if links|render|striptags|trim is not empty %}
-        <dt>{{ "Links"|t }}:</dt>
-        <dd>
-          {{ links }}
-        </dd>
+        <h2>{{ "Links"|t }}</h2>
+
+          <ul>{{ links }}</ul>
```
Comment (buttonwillowsix) [inline templates/stanford_person/stanford-person.html.twig]: @aaroncole this did render out with <LI>. Looks fine on my local.

**Comment by aaroncole** [comment]
Oh, the  {{links}} prints out with <li>s.

**Pull request merged**